test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetClientCertCommandTests.cs

```diff
@@ -265,7 +265,7 @@ public void ClientCertAddCommand_Success_FileCertificateRelative()
             }
         }
 
-        [PlatformFact(Platform.Windows, Platform.Linux, SkipMono = true)]
+        [PlatformFact(Platform.Windows, Platform.Linux, SkipMono = true, Skip = "https://github.com/NuGet/Home/issues/9684")]
```

Comment:
Other tests in this class has `[PlatformFact(Platform.Windows, SkipMono = true)]` attribute. Can we just use that attribute so that the test runs on Windows?

Reply:
This test is failing on Windows intermittently so I want it disabled for all operating systems for now.